### ISSUE TITLE
Creating axios instance not influenced by `axios.defaults`

### ISSUE THREAD

**Issue by 07akioni** [axios/axios#4160]
#### Describe the issue

I'm building a library which uses axios to send requests.

My users also use axios to send requests.

They modified `axios.defaults`.

Their modification breaks my library.

#### Example Code
Code snippet to illustrate your question

```js
// Example code here
```

#### Environment
 - Axios Version [e.g. 0.18.0]
 - Adapter [e.g. XHR/HTTP]
 - Browser [e.g. Chrome, Safari]
 - Browser Version [e.g. 22]
 - Node.js Version [e.g. 13.0.1]
 - OS: [e.g. iOS 12.1.0, OSX 10.13.4]
 - Additional Library Versions [e.g. React 16.7, React Native 0.58.0]

#### Additional context/Screenshots
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by ps2goat** [comment]
Also  related:

I started programming with `const x: Axios = new Axios(<my merged configs>)`, but json was not being transformed! I just did some digging and realized the defaults are never applied within that class like I would expect. I would expect `new Axios(<default config>)` to work the same as `axios.create(<default config>)`

The proposed fix looks like it would at least export the defaults in a way that I could merge them with my other configs.
